econml/dml/dml.py

```diff
@@ -692,7 +713,13 @@ def __init__(self, *,
                  cv=2,
                  mc_iters=None,
                  mc_agg='mean',
-                 random_state=None):
+                 random_state=None,
+                 use_ray=False,
+                 ray_remote_func_options=None,
+                 ):
+        if ray_remote_func_options is None:
```

Comment:
Can you move this logic to within the OrthoLearner fit function, and remove this logic from all subclass __init__ functions. That way we avoid redundant code in all of the subclass __init__ functions and maintain a scikit-learn-like API. If interested in more context, see the "Instantiation" section of this page https://scikit-learn.org/stable/developers/develop.html#apis-of-scikit-learn-objects. 

For instance, imagine a user does the following
```
est = LinearDML(use_ray=some_dict)
est.use_ray = None # user changes their mind about use_ray
est.fit(…)
```
We want the logic of converting None to an empty dict in .fit so we can allow for this kind of behavior.

Reply:
Noted make sense, I will move the redundant code, to fit function within Ortholearner